src/test/com/lambda/MyConsumer.java

```diff
@@ -0,0 +1,7 @@
+package test.com.lambda;
+
+@FunctionalInterface
+public interface MyConsumer<T> {
+    void accept(T t);
+
```

Comment:
you should decide if you leave an empty line after the method in the class or an interface, you can choose any of those options but you should be consistent.